samples/filter_sort_groups.py

```diff
@@ -0,0 +1,91 @@
+####
+# This script demonstrates how to filter groups using the Tableau
+# Server Client.
+#
+# To run the script, you must have installed Python 2.7.9 or later.
+####
+
+
+import argparse
+import getpass
+import logging
+
+import tableauserverclient as TSC
+
+
+def create_example_group(group_name='Example Group', server=None):
```

Comment:
`None` isn't actually a safe default, this should just be a standard required parameter.